exposure-notification/src/main/resources/db/migration/V02.03__alter_configuration_table.sql

```diff
@@ -0,0 +1,2 @@
+alter table en.exposure_configuration add duration_at_attenuation_weights decimal(3,2) array[3];
```

Comment:
How about nulls? I think we should either handle nulls in the fetch-part or add the columns as non-null with default.

Reply:
yep, this is a good point and I actually just bypassed this thing and then forgot it. I'll add not null into these.